### PR TITLE
Add check for 0000_error to script

### PR DESCRIPTION
**Pull request by Marenz**
A user might forget to run scripts/correct_error_ids.py.

### CODE REVIEW

**Comment by leonardoalt** [comment]
I agree this should exist, but not sure it fits the style checker

**Comment by chriseth** [comment]
Agree, we should just run the "fix errors" script and check its return code.

**Comment by Marenz** [comment]
Added it as a ci step now. This is how it looks when it fails: https://circleci.com/gh/ethereum/solidity/340059?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

File: .circleci/config.yml [hunk 1]
```diff
@@ -316,6 +316,18 @@ jobs:
           name: checking shell scripts
           command: ./scripts/chk_shellscripts/chk_shellscripts.sh
 
+  chk_errorcodes:
+    docker:
+      - image: buildpack-deps:eoan
```
Comment (chriseth) [inline .circleci/config.yml]: Maybe you could select an image that already has python3?

File: scripts/correct_error_ids.py [hunk 2]
```diff
@@ -115,19 +118,28 @@ def find_source_files(top_dir):
     return source_file_names
 
 
-def main():
+def main(argv):
+    noconfirm = False
+    opts, args = getopt.getopt(argv,"",["noconfirm"])
```
Comment (chriseth) [inline scripts/correct_error_ids.py]:
```suggestion
    opts, args = getopt.getopt(argv, "", ["noconfirm"])
```

File: scripts/correct_error_ids.py [hunk 3]
```diff
@@ -115,19 +118,28 @@ def find_source_files(top_dir):
...
+
+    for opt, arg in opts:
+        if opt == '--noconfirm':
```
Comment (chriseth) [inline scripts/correct_error_ids.py:126]: Shouldn't this err on invalid options?

Comment (Marenz) [reply scripts/correct_error_ids.py:126]: It does:
```
 marenz@edox  ~/eth/solidity   extend-check-srcipt  scripts/correct_error_ids.py --wrong
Traceback (most recent call last):
  File "scripts/correct_error_ids.py", line 170, in <module>
    main(sys.argv[1:])
  File "scripts/correct_error_ids.py", line 123, in main
    opts, args = getopt.getopt(argv,"",["noconfirm"])
  File "/usr/lib/python3.8/getopt.py", line 93, in getopt
    opts, args = do_longs(opts, args[0][2:], longopts, args[1:])
  File "/usr/lib/python3.8/getopt.py", line 157, in do_longs
    has_arg, opt = long_has_args(opt, longopts)
  File "/usr/lib/python3.8/getopt.py", line 174, in long_has_args
    raise GetoptError(_('option --%s not recognized') % opt, opt)
getopt.GetoptError: option --wrong not recognized

```

**Comment by Marenz** [comment]
Using python image now
fixed white space issues

**Review by chriseth — APPROVED** [review]

**Pull request merged**